infra/sync/build.gradle.kts

```diff
@@ -0,0 +1,46 @@
+plugins {
+    id("simprints.infra")
+}
+
+android {
+    namespace = "com.simprints.infra.sync"
+
+
+    buildTypes {
+        getByName("release") {
+            buildConfigField("long", "PROJECT_DOWN_SYNC_WORKER_INTERVAL_MINUTES", "60L")
+            buildConfigField("long", "DEVICE_DOWN_SYNC_WORKER_INTERVAL_MINUTES", "30L")
+            buildConfigField("long", "IMAGE_UP_SYNC_WORKER_INTERVAL_MINUTES", "60L")
+            buildConfigField("long", "FIRMWARE_UPDATE_WORKER_INTERVAL_MINUTES", "1440L")
+            buildConfigField("long", "EVENT_SYNC_WORKER_INTERVAL_MINUTES", "60L")
+        }
+        getByName("staging") {
+            buildConfigField("long", "PROJECT_DOWN_SYNC_WORKER_INTERVAL_MINUTES", "15L")
+            buildConfigField("long", "DEVICE_DOWN_SYNC_WORKER_INTERVAL_MINUTES", "15L")
+            buildConfigField("long", "IMAGE_UP_SYNC_WORKER_INTERVAL_MINUTES", "15LL")
+            buildConfigField("long", "FIRMWARE_UPDATE_WORKER_INTERVAL_MINUTES", "15LL")
```

Comment:
`15LL` - is this a typo in `IMAGE_UP_SYNC_WORKER_INTERVAL_MINUTES`, `EVENT_SYNC_WORKER_INTERVAL_MINUTES` and `FIRMWARE_UPDATE_WORKER_INTERVAL_MINUTES`?